install.ts

```diff
@@ -0,0 +1,81 @@
+import { parse } from "https://deno.land/std@0.136.0/flags/mod.ts";
+import { red } from "https://deno.land/std@0.136.0/fmt/colors.ts";
+
+/*
+ * Prompt for a response
+ */
+async function prompt(message = "") {
+  const buf = new Uint8Array(1024);
+  await Deno.stdout.write(new TextEncoder().encode(message + ": "));
+  const n = <number> await Deno.stdin.read(buf);
+  return new TextDecoder().decode(buf.subarray(0, n)).trim();
+}
+
+export async function checkVersion(version: string): Promise<string> {
+  console.log("Looking up latest version...");
+
+  const versionMetaUrl = "https://cdn.deno.land/strawman/meta/versions.json";
```

Comment:
```suggestion
  const VERSION_META_URL = "https://cdn.deno.land/strawman/meta/versions.json";
```

Also, this constant should live outside the `checkVersion` function (ergo: in module scope).